python/example_load_gen.py

```diff
@@ -0,0 +1,307 @@
+    ) -> str:
+        return f"""
+    count: {histogram.total_count}
+      min: {histogram.min_value}
+      p50: {histogram.get_value_at_percentile(50)}
+      p90: {histogram.get_value_at_percentile(90)}
+      p99: {histogram.get_value_at_percentile(99)}
+      max: {histogram.max_value}         
+"""
+
+    @staticmethod
+    def get_elapsed_millis(start_time: float) -> int:
+        end_time = perf_counter_ns()
+        result = round((end_time - start_time) / 1e6)
+        return result
+
+
+PERFORMANCE_INFORMATION_MESSAGE = """
+Thanks for trying out our basic python load generator!  This tool is
+included to allow you to experiment with performance in your environment
+based on different configurations.  It's very simplistic, and only intended
+to give you a quick way to explore the performance of the Momento client
+running on a single python process.
+
+Note that because python has a global interpreter lock, user code runs on
+a single thread and cannot take advantage of multiple CPU cores.  Thus, the
+limiting factor in request throughput will often be CPU.  Keep an eye on your CPU
+consumption while running the load generator, and if you reach 100%
+of a CPU core then you most likely won't be able to improve throughput further
+without running additional python processes.
+
+CPU will also impact your client-side latency; as you increase the number of
+concurrent requests, if they are competing for CPU time then the observed
+latency will increase.
+
+Also, since performance will be impacted by network latency, you'll get the best
+results if you run on a cloud VM in the same region as your Momento cache.
+
+Check out the configuration settings at the bottom of the 'example_load_gen.py' to
+see how different configurations impact performance.
+
+If you have questions or need help experimenting further, please reach out to us!
+"""
+
+
+async def main(
+        log_level: int,
+        request_timeout_ms: int,
+        cache_item_payload_bytes: int,
+        number_of_concurrent_requests: int,
+        total_number_of_operations_to_execute: int
+) -> None:
+    initialize_logging(log_level)
+    load_generator = BasicPythonLoadGen(
+        request_timeout_ms=request_timeout_ms,
+        cache_item_payload_bytes=cache_item_payload_bytes,
+        number_of_concurrent_requests=number_of_concurrent_requests,
+        total_number_of_operations_to_execute=total_number_of_operations_to_execute
+    )
+    await load_generator.run()
+
+
+load_generator_options = dict(
+    #
+    # This setting allows you to control the verbosity of the log output during
+    # the load generator run. Available log levels are TRACE, DEBUG, INFO, WARN,
+    # and ERROR.  DEBUG is a reasonable choice for this load generator program.
+    #
+    log_level=logging.DEBUG,
+    #
+    # Configures the Momento client to timeout if a request exceeds this limit.
+    # Momento client default is 5 seconds.
+    #
+    request_timeout_ms=5 * 1_000,
```

Comment:
should we up this default higher since devs maybe running on laptop? Maybe just need to remember to come back and adjust here after convo on timeout values to default to in sdk.

Reply:
yeah in this case i'm expecting users would look at this batch of settings if they were seeing output that they weren't happy with, so they could tweak it themselves.  Happy to bump it up if we bump up the SDK default.